### DISCUSSION TITLE
[Question] - Mounted kubernetes secrets volume

### DISCUSSION THREAD

**Discussion by dazinator** [mcrio/Configuration.Provider.Docker.Secrets#3] [Q&A] [answered]
Hello,

Do you know if this library would also handle a mounted volume containing Key Vault secret files, based on the setup for kubernets described here:

https://docs.microsoft.com/en-us/azure/key-vault/general/key-vault-integrate-kubernetes

I am guessing that it `should` as its just a set of files? Would be interested to know if anyone has tried this.

**Comment by mcrio** [comment, marked as answer]
You should be good to go. Just specify the correct secrets directory path. The default one is `/run/secrets`.
https://github.com/mcrio/Configuration.Provider.Docker.Secrets/blob/ad43948da6d1f5c18a68f4641f4dfd462042942a/Configuration.Provider.Docker.Secrets/DockerSecretsConfigurationExtension.cs#L13
